cypress/integration/SettingsScreen.ts

```diff
@@ -7,5 +7,23 @@ describe('SettingsScreen', () => {
     cy.contains('Settings')
 
     cy.contains('Clear Data')
+    cy.contains('About')
+  })
+
+  it('can navigate to about page and back', () => {
+    cy.visit('/')
+
+    cy.contains('Settings').click()
+
+    cy.contains('About').click()
+
+    cy.contains('About')
+
+    cy.contains('Application Version')
+    cy.contains('Build Version')
```

Comment:
Sadly we can not assert on the actual version numbers in cypress as the build does not have a application or build version locally.